src/StatsBase.jl

```diff
@@ -156,7 +156,11 @@ module StatsBase
     residuals,
     r2,
     r²,
-    model_response
+    model_response,
+
+    ## comparison tests for statistical models
+    testfit # Pairwise sequential tests of model fit.
```

Comment:
Just put this among the other modeling functions, it doesn't need a separate list.